src/test/java/org/exmple/utility/UtilityMethodsTest.java

```diff
@@ -96,4 +93,34 @@ public void countEven_shouldThrowNullPointerException() {
 
         assertTrue(actualMessage.contains(expectedMessage));
     }
+
+    @Test
+    public void copy_shouldReturnCopiedArray() {
+        int[] array = {1, 2, 3, -7};
+        boolean isEquals = true;
+
+        int[] expectedArray = {1, 2, 3, -7};
+        int[] actualArray = UtilityMethods.copy(array);
+
+        for (int i = 0; i < array.length; i++) {
+            if (expectedArray[i] != actualArray[i]) {
```

Comment:
I think it is better to have an assertion here. So it looks like: `assertEquals(expectedArray[i], actualArray[i])`. Then we won't need to have the `isEquals` variable